packages/strapi/lib/commands/uninstall.js

```diff
@@ -21,7 +21,7 @@ module.exports = async (plugins, { deleteFiles }) => {
   const loader = ora();
   const dir = process.cwd();
 
-  const pluginArgs = plugins.map(name => `strapi-plugin-${name}`);
+  const pluginArgs = plugins.map((name) => `strapi-plugin-${name}`);
```

Comment:
If you wold like to avoid Arrow Function Parentheses I suggest to add in `.prettierrc.js` file:
```
arrowParens: 'avoid',
```